src/snowflake/snowpark/mock/_connection.py

```diff
@@ -795,6 +795,9 @@ def get_result_query_id(self, plan: SnowflakePlan, **kwargs) -> str:
         # get the iterator such that the data is not fetched
         result_set, _ = self.get_result_set(plan, to_iter=True, **kwargs)
         return result_set["sfqid"]
+    
+    def create_coprocessor(self):
```

Comment:
would we add in the future any functionality to this? Or is this a simple stub? In any case maybe a short comment to  address either 1.) what needs to get added here or 2.) that this should not be removed, but left because of ... would help :)

Reply:
Done.